specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-02-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+      "in": "query",
+      "required": false,
+      "description": "A filter which allows the get entities call to focus on a particular group (i.e. \"$filter=name eq 'groupName'\")",
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "CreateOrUpdateManagementGroupSettingsRequestParameter": {
+      "name": "CreateTenantSettingsRequest",
+      "in": "body",
+      "required": true,
+      "description": "Tenant level settings request parameter.",
+      "schema": {
+        "$ref": "#/definitions/CreateOrUpdateSettingsRequest"
+      },
+      "x-ms-parameter-location": "method"
+    },
+    "CheckNameAvailabilityParameter": {
+      "name": "checkNameAvailabilityRequest",
+      "in": "body",
+      "required": true,
+      "description": "Management group name availability check parameters.",
+      "schema": {
+        "$ref": "#/definitions/CheckNameAvailabilityRequest"
+      },
+      "x-ms-parameter-location": "method"
+    },
+    "SkipParameter": {
+      "name": "$skip",
+      "in": "query",
+      "required": false,
+      "type": "integer",
+      "description": "Number of entities to skip over when retrieving results. Passing this in will override $skipToken.",
+      "x-ms-parameter-location": "method"
+    },
+    "TopParameter": {
+      "name": "$top",
+      "in": "query",
+      "required": false,
+      "type": "integer",
+      "description": "Number of elements to return when retrieving results. Passing this in will override $skipToken.",
+      "x-ms-parameter-location": "method"
+    },
+    "SkipTokenParameter": {
+      "name": "$skiptoken",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "description": "Page continuation token is only used if a previous operation returned a partial result. \nIf a previous response contains a nextLink element, the value of the nextLink element will include a token parameter that specifies a starting point to use for subsequent calls.\n",
+      "x-ms-parameter-location": "method"
+    },
+    "CacheControlHeader": {
+      "name": "Cache-Control",
+      "in": "header",
+      "default": "no-cache",
+      "description": "Indicates whether the request should utilize any caches. Populate the header with 'no-cache' value to bypass existing caches.",
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "EntityViewParameter": {
+      "name": "$view",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "enum": [
+        "FullHierarchy",
+        "GroupsOnly",
+        "SubscriptionsOnly",
+        "Audit"
+      ],
+      "x-ms-enum": {
+        "name": "EntityViewParameterType",
+        "modelAsString": true
+      },
+      "x-ms-parameter-location": "method",
+      "description": "The view parameter allows clients to filter the type of data that is returned by the getEntities call."
+    },
+    "ApplicationIdParameter": {
+      "name": "applicationId",
```

Comment:
>applicationId [](http://example.com/codeflow?start=15&length=13)

Is this really an ID like a guid, or is it a name chosen by the client? If the latter, I'd recommend calling it applicationName. If it has an enforced format like guid, then applicationId is appropriate, but you should mention the format in the description. You can also enforce the format as uuid, or using a regex.